contrib/api-stress-test/readme.md

```diff
@@ -105,8 +105,16 @@ locust_worker_number
 
 #### Running playbooks
 
+##### Start Virtual-kubelet and locust
 ```
-ansible-playbook -i host.yml stress.yml [-e "@path/to/env.yml"]
+ansible-playbook -i host.yml stress-part1.yml [-e "@path/to/env.yml"]
+```
+
+This playbooks will take some time to be complated. Because vk will be added into the cluster one by one. Please ensure all the vk is added into the cluster, then go to the next steps.
```

Comment:
Can we speed up the process by setting `-f <parallel-number>` option in ansible?

Reply:
No, we can't. It‘s in vk code.